chart/values.schema.json

```
@@ -711,6 +711,58 @@
                 }
             ]
         },
+        "enableBuiltInSecretEnvVars": {
+            "description": "Uses built-in secret values set as environment variables passed to Airflow. You should supply corresponding environment variables as ``extraEnv`` variables if you disable them here.",
+            "type": "object",
+            "x-docsSection": "Airflow",
+            "properties": {
+                "AIRFLOW__CORE_FERNET_KEY": {
+                    "description": "Enable Airflow fernet key variable read from .Values.fernetKeySecretName",
```

I'm a little torn on the pattern of mentioning `.Values.fernetKeySecretName`, because by default that is null and the secret name is defaulted to `{{ .Release.Name }}-fernet-key` in this specific case. I wonder if this would be better?

```suggestion
                    "description": "Enable Airflow fernet key variable read from fernet key secret",
```

---

I am all for being explicit so I added longer explanation. It does not hurt.